backend/src/routers/siteAdminRouter.ts

```diff
@@ -200,13 +196,64 @@ siteAdminRouter.delete('/orgs/:id', async (req, res) => {
 	res.status(200).json({ message: 'Organization deleted' });
 });
 
+/**
+ * @api {post} /api/v1/admin/orgs/:id Update organization
+ * @apiPermission siteAdmin
+ * @apiName UpdateOrganization
+ * @apiGroup SiteAdmin
+ * @apiDescription Updates an organization
+ *
+ * @apiBody {String} name The name of the organization
+ * @apiBody {String[]} gammaSuperGroups The gamma super groups to link to the organization
+ * @apiBody {Number} addGammaAsOrgAdmin Whether to add the gamma super groups members as organization admins
+ *
+ * @apiSuccess {String} message The organization was updated
+ *
+ * @apiSuccessExample Success-Response:
+ *  HTTP/1.1 200 OK
+ *
+ *  {
+ *      "message": "Organization updated"
+ *  }
+ *
+ * @apiUse ZodError
+ *
+ */
+siteAdminRouter.put(
+	'/orgs/:id',
+	validateRequestBody(addOrEditOrganizationSchema),
+	async (req, res) => {
+		const org = await getOrganization(req.params.id);
+		if (!org) {
+			return res.status(404).json({ message: 'Organization not found' });
+		}
+
+		var validationErrors: ErrorProperty[] = await validateGammaGroups(
```

Comment:
That `var` scares me, should probably be `let` // Sonic
```suggestion
		let validationErrors: ErrorProperty[] = await validateGammaGroups(
```

Reply:
Written at 01:00. I did not remember which to use...